src/Uno.UI/UI/Xaml/FontFamily.wasm.cs

```diff
@@ -1,58 +1,48 @@
 ﻿#if __WASM__
+#nullable enable
+
 using System;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using Uno;
 
 namespace Windows.UI.Xaml.Media
 {
 	public partial class FontFamily
 	{
-		partial void Init(string fontName)
-		{
-			ParsedSource = ParseFontFamilySource(fontName);
-		}
+		private FontLoader _loader;
+
+		partial void Init(string fontName) => _loader = FontLoader.GetLoaderForFontFamily(this);
+
+		/// <summary>
+		/// Contains the font-face name to use in CSS.
+		/// </summary>
+		internal string CssFontName => _loader.CssFontName;
 
 		/// <summary>
-		/// Contains the parsed font family for use in WASM
-		/// (matches CSS @font-face's font-family)
+		/// Use this to launch the loading of a font before it is actually required to
+		/// minimize loading time and prevent potential flicking.
 		/// </summary>
-		internal string ParsedSource { get; private set; }
+		public static void Preload(FontFamily family) => family._loader.LaunchLoading();
 
-		private string ParseFontFamilySource(string familyName)
+		/// <summary>
+		/// Use this to launch the loading of a font before it is actually required to
+		/// minimize loading time and prevent potential flicking.
+		/// </summary>
+		public static void Preload(string familyName) => Preload(new FontFamily(familyName));
```

Comment:
I think that these methods need to be `internal` and then need to be made `public` using an API inside `Uno.UI.Runtime.WebAssembly`, right @jeromelaban? Otherwise they will not actually be accessible (because the magic with reference API, only things that are netstdref are actually visible). I hit the same problem when I implemented `CookieManager` which is WASM specific - it was usable from Samples app (as it references Uno.UI.Wasm directly), but then not usable in actual apps, and had to move it to `Uno.UI.Runtime.WebAssembly` for that reason.

Reply:
Those methods (even with the newer changed signatures) are accessible from the main app, but indeed won't be from a library. The point is for these methods to be called during startup in the main project, so if we need to move those to the runtime assembly, we'll be able to do so in the future.